debian/gnome-software-dev.install

```diff
@@ -1,2 +1,3 @@
 usr/include/gnome-software/
+usr/lib/*/libgnomesoftware.a
```

Comment:
I guess this is a product of [this commit](https://gitlab.gnome.org/GNOME/gnome-software/-/commit/d3fae3b356edc59b53e476de19549c118a6bdc78), and we don't need or want to support out-of-tree plugins, so that's why we're putting it in gnome-software-dev instead of gnome-software?

Reply:
Might be worth explaining that in the commit message

Reply:
We’re putting it in gnome-software-dev rather than gnome-software because aiui `.a` static libraries always go in the `-dev` package. They’re only needed when building things which depend on them, not when running those things.

I’ve mentioned it in the commit message, thanks